lib/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+		minSdkVersion 14
+		targetSdkVersion 28
+		testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+		externalNativeBuild {
+            cmake {
```

Comment:
I think here there is some mismatch between tabs and spaces. I think that `cmake` should be more indented than `externalNativeBuild`, rather than less indented.

Reply:
@xanscale this has not been done